tests/testthat/test-old_new.R

```diff
@@ -0,0 +1,35 @@
+context('Old new tests')
+# Verify that hector output for the canonical rcps matches output from older version. This "bit for bit"
+# comparison of the output it intended to help preserve Hector data during minor devlopment.
```

Comment:
"...to help guarantee that Hector's outputs don't change during minor development."